rangeset/fuzz/Cargo.toml

```diff
@@ -13,9 +13,6 @@ libfuzzer-sys = { version = "0.4", features = ["arbitrary-derive"] }
 [dependencies.rangeset]
 path = ".."
 
-[profile.release]
-debug = 1
-
```

Comment:
has been ported from her, see https://stackoverflow.com/questions/45794917/specific-profiles-for-workspace-members